spring/src/main/java/org/axonframework/spring/config/CommandHandlerSubscriber.java

```diff
@@ -33,6 +33,7 @@
  *
  * @author Allard Buijze
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.